src/core/editform/qgsattributeeditorelement.cpp

```diff
@@ -86,9 +97,82 @@ QgsAttributeEditorElement *QgsAttributeEditorElement::create( const QDomElement
     else
       newElement->setShowLabel( true );
 
+    // Label font and color
+    LabelStyle style;
+    style.readXml( element );
+    newElement->setLabelStyle( style );
+
     newElement->loadConfiguration( element, layerId, context, fields );
   }
 
   return newElement;
 }
 
+
+void QgsAttributeEditorElement::LabelStyle::readXml( const QDomNode &node )
+{
+  QDomElement element { node.firstChildElement( QStringLiteral( "labelStyle" ) ) };
+
+  if ( ! element.isNull() )
+  {
+
+    // Label font and color
+    if ( element.hasAttribute( QStringLiteral( "labelColor" ) ) )
+    {
+      color = QgsSymbolLayerUtils::decodeColor( element.attribute( QStringLiteral( "labelColor" ) ) );
+    }
+
+    QFont newFont;
+    QgsFontUtils::setFromXmlChildNode( newFont, element, QStringLiteral( "labelFont" ) );
+
+    // Font properties
+    if ( element.hasAttribute( QStringLiteral( "fontBold" ) ) )
+    {
+      newFont.setBold( element.attribute( QStringLiteral( "fontBold" ) ) == QChar( '1' ) );
+    }
+    if ( element.hasAttribute( QStringLiteral( "fontItalic" ) ) )
+    {
+      newFont.setItalic( element.attribute( QStringLiteral( "fontItalic" ) ) == QChar( '1' ) );
+    }
+    if ( element.hasAttribute( QStringLiteral( "fontUnderline" ) ) )
+    {
+      newFont.setUnderline( element.attribute( QStringLiteral( "fontUnderline" ) ) == QChar( '1' ) );
+    }
+    if ( element.hasAttribute( QStringLiteral( "fontStrikethrough" ) ) )
+    {
+      newFont.setStrikeOut( element.attribute( QStringLiteral( "fontStrikethrough" ) ) == QChar( '1' ) );
+    }
+
+    font = newFont;
+
+    if ( element.hasAttribute( QStringLiteral( "overrideLabelColor" ) ) )
+    {
+      overrideColor = element.attribute( QStringLiteral( "overrideLabelColor" ) ) == QChar( '1' );
+    }
+
+    if ( element.hasAttribute( QStringLiteral( "overrideLabelFont" ) ) )
+    {
+      overrideFont = element.attribute( QStringLiteral( "overrideLabelFont" ) ) == QChar( '1' );
+    }
+  }
+}
+
+QDomElement QgsAttributeEditorElement::LabelStyle::writeXml( QDomDocument &document ) const
+{
+  QDomElement elem {  document.createElement( QStringLiteral( "labelStyle" ) ) };
+  elem.setAttribute( QStringLiteral( "labelColor" ), QgsSymbolLayerUtils::encodeColor( color ) );
+  elem.appendChild( QgsFontUtils::toXmlElement( font, document, QStringLiteral( "labelFont" ) ) );
+  elem.setAttribute( QStringLiteral( "overrideLabelColor" ), overrideColor ? QChar( '1' ) : QChar( '0' ) );
+  elem.setAttribute( QStringLiteral( "overrideLabelFont" ), overrideFont ? QChar( '1' ) : QChar( '0' ) );
+  // Font properties
+  elem.setAttribute( QStringLiteral( "fontBold" ), font.bold() ? QChar( '1' ) : QChar( '0' ) );
```

Comment:
Any particular reason not to add these to QgsFontUtils::toXmlElement?

Reply:
not really

Reply:
Done with https://github.com/qgis/QGIS/pull/48353/commits/ee204691e2d9504a1f27dce22e9fe60604308f3d